### PR TITLE
update gitlab docs to use 2023 saml

### PR DESCRIPTION
**Pull request by timothy-spencer**
Also add a missing line so that the cert will be proper.

### CODE REVIEW

**Review by zachmargolis — APPROVED** [review]
LGTM

File: _articles/platform-gitlab.md [hunk 1]
```diff
@@ -121,14 +121,15 @@ GitLab leverages Omniauth to allow users to sign in using a variety of services,
 openssl req -nodes -x509 -days 365 -newkey rsa:2048 -keyout private.pem -out public.crt
 ```
 
-1. Grab the IDP sandbox signing certificate from <https://developers.login.gov/saml/> and get its fingerprint:
+1. Grab the IDP sandbox signing certificate from <https://developers.login.gov/saml/> and get its fingerprint (use secure.login.gov instead of idp.int.identitysandbox.gov if you are doing this for gitlab prod/staging):
 ```
-curl -s https://idp.int.identitysandbox.gov/api/saml/metadata2021 \
+curl -s https://idp.int.identitysandbox.gov/api/saml/metadata2023 \
```
Comment (zachmargolis) [inline _articles/platform-gitlab.md:126]: if we wanted, we could use jekyll to magically always use the current year?
```suggestion
curl -s https://idp.int.identitysandbox.gov/api/saml/metadata{{ site.time | date: '%Y' }}
 \
```

Comment (timothy-spencer) [reply _articles/platform-gitlab.md:126]: maaaaybe?  I've never done that.  Is there ever any time in 202X where the 202X saml cert won't be available?

Comment (zachmargolis) [reply _articles/platform-gitlab.md]: Ideally our process would be smooth like that, but I'm not sure how smooth it actually is

**Pull request merged**